presentation/src/main/java/com/stop/ui/routedetail/RouteDetailFragment.kt

```diff
@@ -34,19 +37,59 @@ class RouteDetailFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        initTMap()
         initView()
+        setRecyclerView()
+    }
+
+    override fun alertTMapReady() {
+        routeResultViewModel.itinerary.value?.let {
+            tMap.drawRoutes(it.routes)
+        }
     }
 
     private fun initBinding() {
         binding.lifecycleOwner = viewLifecycleOwner
         binding.routeResultViewModel = routeResultViewModel
+        binding.itinerary = routeResultViewModel.itinerary.value
+    }
+
+    private fun initTMap() {
+        tMap = RouteDetailTMap(requireActivity(), this)
+        tMap.init()
+
+        binding.layoutContainer.addView(tMap.tMapView)
     }
 
     private fun initView() {
-        //TODO 임시 화면이동 로직
-        binding.root.setOnClickListener {
-            findNavController().navigate(R.id.action_routeDetailFragment_to_alarmSetting)
+        binding.layoutDrawer.openDrawer(GravityCompat.START)
+
+        binding.imageViewArrowDrawer.setOnClickListener {
+            binding.layoutDrawer.openDrawer(GravityCompat.START)
         }
+
+        binding.routeDetailDrawer.viewAlarm.setOnClickListener {
+            binding.root.findNavController().navigate(R.id.action_routeDetailFragment_to_alarmSetting)
+        }
+
+        binding.imageViewClose.setOnClickListener {
+            findNavController().apply {
+                setGraph(R.navigation.nav_graph)
+                popBackStack(R.id.action_global_mapFragment, false)
```

Comment:
```suggestion
                popBackStack(R.id.action_global_mapFragment, false)
                requireActivity().viewModelStore.clear()
```
popBackStack을 사용해서 BackStack에 MapFragment가 2개 존재하지 않도록 해야합니다.
또한 RouteFragment에 있는 ActivityViewModels를 ViewModelStore를 초기화해서 지워줘야합니다.
그렇지 않으면 RouteViewModel에 있는 출발지, 도착지, 경로 데이터가 그대로 살아있게 됩니다.